exampleSite/config.toml

```diff
@@ -61,6 +61,10 @@ googleAnalytics = ""
   #[params.contact.captcha]
   #  sitekey = ""
 
+  # [params.navbar]
+  #   Force navbar to be displayed even if no menu items are available
+  #   force_display = ture
```

Comment:
typo

Reply:
Fixed in 9104649575b9ec01f5572cd49201775b0f7ddd97